plugins/GSdx/GSRenderer.cpp

```diff
@@ -293,7 +306,7 @@ bool GSRenderer::Merge(int field)
 			m_dev->ShadeBoost();
 		}
 
-		if (m_shaderfx)
+		if(m_shaderfx)
 		{
```

Comment:
Please adopt this style also for the ``if()`` checks present on line [64](https://github.com/FlatOutPS2/pcsx2/blob/fe6bb36ee26e8ae6a5229af0a0d834b87e220287/plugins/GSdx/GSRenderer.cpp#L64) , [375](https://github.com/FlatOutPS2/pcsx2/blob/fe6bb36ee26e8ae6a5229af0a0d834b87e220287/plugins/GSdx/GSRenderer.cpp#L375) and [377](https://github.com/FlatOutPS2/pcsx2/blob/fe6bb36ee26e8ae6a5229af0a0d834b87e220287/plugins/GSdx/GSRenderer.cpp#L377).

Turtleli would feel sad if he saw those inconsistent placement of ``if()`` checks :/